### PR TITLE
BOM-1231

### PR DESCRIPTION
**Pull request by UsamaSadiq**
Updated social auth app django for Django2.0 support

### CODE REVIEW

File: requirements/github.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Forks and other dependencies not yet on PyPI
+
+# Forked to get Django 2.2 support from unreleased master branch from social-app-django repo.
+# This can be removed once an official social-auth-app-django Pypi release with Django 2.2 support is available in the future.
+-e git+https://github.com/python-social-auth/social-app-django.git@c00d23c2b45c3317bd35b15ad1b959338689cef8#egg=social-auth-app-django
```
Comment (Ayub-Khan) [inline requirements/github.in]: Remove '-e'. No need.

**Review by Ayub-Khan — CHANGES_REQUESTED** [review]

File: requirements/github.in (hunk 1 above)
Comment (UsamaSadiq) [reply requirements/github.in]: Updated as per suggestion.

**Review by mraarif — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Review by jmbowman — APPROVED** [review]

**Review by Dillon-Dumesnil — APPROVED** [review]
Do you think it would be helpful to create a ticket that we can check in on in sixth months or so to see if a release has happened? My only concern here is that we forget about this line of code and it just stays this way. Creating a ticket is not necessary and only intended as a potential help, but if we don't think it would be helpful, we do not need to pursue it.

**Comment by Dillon-Dumesnil** [comment]
Also it looks like Ayub-Khan will either need to give an approval on this PR or that review needs to be dismissed before this can merge as it is currently blocking merging.

**Review by Ayub-Khan — APPROVED** [review]

**Pull request merged**